datasetinsights/commands/download.py

```diff
@@ -80,51 +74,19 @@ def convert(self, value, param, ctx):
     ),
 )
 @click.option(
-    "--dataset-version",
-    type=click.STRING,
-    default=const.DEFAULT_DATASET_VERSION,
-    help=(
-        "The dataset version. This only applies to some dataset "
-        "where different versions are available."
-    ),
-)
-@click.option(
-    "--project-id",
-    type=click.STRING,
+    "--access-token",
+    type=str,
     default=None,
-    help=(
-        "Unity project-id used for Unity Simulation runs. This will override "
-        "synthetic datasets source-uri for Unity Simulation."
-    ),
-)
-@click.option(
-    "--run-execution-id",
-    type=click.STRING,
-    default=None,
-    help=(
-        "Unity Simulation run-execution-id. This will override synthetic "
-        "datasets source-uri for Unity Simulation."
-    ),
-)
-@click.option(
-    "--auth-token",
-    type=click.STRING,
-    default=None,
-    help=(
-        "Unity Simulation auth token. This will override synthetic "
-        "datasets source-uri for Unity Simulation."
-    ),
+    help="Access-token to be used to authenticate"
+    " to unity simulation for downloading the dataset",
```

Comment:
[nit] CAPS for `Unity Simulation`. The description is a little harder to read (with three `to`). Maybe we should rephrase it a little bit.